src/widgets/auth/views/signupViewComponent.jsx

```diff
@@ -43,13 +46,35 @@ const UserAggreementStyle = withTheme(styled.div`
 `);
 
 export default class SignupView extends React.Component {
+    state = {
+        blacklist: ['chocolatine']
```

Comment:
Shouldn't the state be initialized to an empty array?

Reply:
`chocolatine` is an eternally taboo word.

Reply:
It actually gets emptied on first load. We can remove the humor if necessary :-)

Reply:
It's a funny idea, unfortunately, I fear that it might confuse developers (like me ><).